### ISSUE TITLE
ScreenOff missing from Play Store

### ISSUE THREAD

**Issue by gene1wood** [jsalatas/ScreenOff#2]
It looks like the ScreenOff app is now missing from the Play Store

https://play.google.com/store/apps/details?id=gr.ictpro.jsalatas.screenoff

**Comment by jsalatas** [comment]
Yeah! I know! Unfortunately the constant change of requirements in google play, make it impossible for me to keep up.

You may have a chance of building it from the source code, but keep in mind that I'm not actively maintaining it any more. Maybe I should remove the play store link and archive the project :\

**Comment by gene1wood** [comment]
Totally understand. My workaround was to extract the apk from my old phone and install it on my new phone.

I did this by running

```
adb shell pm path gr.ictpro.jsalatas.screenoff
```

which returned the path to the apk on my phone

and then
```
adb pull /data/app/~~8gOLXSp_8EwasyIXlARPyg==/gr.ictpro.jsalatas.screenoff-4rmO59u35UldR_QFrwfUDQ==/base.apk
```

to pull the apk.

You could add the apk to the release section or the repo for future folks to fetch if they want to use it.

Thank you for this app though, it's been really useful over the years.

**Comment by jsalatas** [comment]
Thanks! 

I'm thinking of publishing it to f-droid if I find some time.

**Comment by rsaurav12** [comment]
please publish it to f droid
